routes/update-routes.js

```diff
@@ -123,6 +127,34 @@ module.exports = (app) => {
     }
   );
 
+  //This route sets geolocation of a user in the profile
+  app.put(
+    "/api/profile/location/geo",
+    passport.authenticate("jwt", { session: false }),
+    (req, res) => {
+      const userIP = req.headers["x-forwarded-for"] || req.ip;
+      const info = lookup.get(userIP);
+
+      if (!info || !info.country || !info.city || !info.location) {
```

Comment:
why put check on info.country, ect  if there is a check if the whole info object exists? Like if info is null, info.country would not exist , or am I missing something ? lol